cobalt/black_box_tests/tests/allow_eval.py

```diff
@@ -25,9 +25,9 @@ class AllowEvalTest(black_box_tests.BlackBoxTestCase):
   """Ensure that client page can use eval() when CSP is missing."""
 
   def test_allow_eval(self):
-
     with ThreadedWebServer(binding_address=self.GetBindingAddress()) as server:
       url = server.GetURL(file_name='testdata/allow_eval.html')
 
-      with self.CreateCobaltRunner(url=url) as runner:
+      with self.CreateCobaltRunner(
+          url=url, web_server_port=server.GetPort()) as runner:
```

Comment:
passing `web_server_port` here requires modifying _all_ files. Is there perhaps another way how this argument can be passed down to the runner ? It already carries `launcher_params` and `target_params`

Reply:
The only option I see here is to extract the web server's creation and destruction from each test level to the test suite's initialization. I think that might be a good change. But we will definitely want a dedicated PR for such big modification.